packages/ui-kit/src/components/rss/rss-feeds.tsx

```diff
@@ -72,11 +72,12 @@ const fetcher = async (args: string[]) => {
 };
 
 const transformData = (data: FlatRssEntry[][]) => {
-  // First, we need to get the oldest release note from each feed,
+  // First, we need to get the oldest release note from feeds with at least 10 entries,
```

Comment:
Please point to the place in the code where the 10 is defined and make a strong statement that it must by synced if that changes.   Same in the other direction in the place where the 10 is defined.   
I think we can't make it a central constant because it's in ui-kit vs. docs-kit but a comment should be good enough.

Reply:
That is the only place related to the release notes with a hardwired 10 I was able to find...but I guess that's not the place you had in mind?
https://github.com/commercetools/commercetools-docs-kit/blob/main/packages/gatsby-theme-docs/src/components/release-notes-filter-dates.js#L56

Reply:
Hier ist der feed kodiert mit 10 hardwired:  https://github.com/commercetools/commercetools-docs-kit/blob/main/packages/gatsby-theme-docs/gatsby-config.mjs#L298  

Die 10 könntest du sogar als template reinnehmen aber wie gesagt wäre das nervig weil ui-kit<->docs-kit abhängigkeit.

Reply:
Ah in der query, verstehe 👍🏼 dann verlinke ich die beiden Stellen noch.